### PR TITLE
Refactor: bridge panel

### PR DESCRIPTION
**Pull request by chezhe**
- [x] Optimize BridgePanel component
- [x] Add EVM compatible feature

### CODE REVIEW

File: src/components/Bridge/TokenInput.tsx [hunk 1]
```diff
@@ -15,29 +15,153 @@ import {
...
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+  const isNear = chain === "NEAR"
+  const onSetMax = () => {
+    if (!isNear && tokenAsset?.assetId === undefined) {
+      onUpdateAmount(
+        balance?.sub(0.1).gt(ZERO_DECIMAL) ? balance?.sub(0.1).toString() : ""
+      )
+    } else {
+      onUpdateAmount(balance?.toString() || "")
```
Comment (madwiki) [inline src/components/Bridge/TokenInput.tsx:107]: 0.1 native token for EVM appchain gas fee is too much. BaseFee in moonbeam appchain right now is a constant named gasPrice. 200,000 * 10 * gasPrice is enough, for barnacle-evm, it is only 0.0002.

File: src/views/Bridge/BridgePanel.tsx [hunk 2]
```diff
@@ -790,31 +596,28 @@ export const BridgePanel: React.FC = () => {
...
+                  : !to
                   ? "Input Target Account"
-                  : isInvalidTargetAccount || targetAccountNeedDepositStorage
+                  : targetAccountNeedDepositStorage
                   ? "Invalid Target Account"
                   : !collectible
                   ? !amount
                     ? "Input Amount"
-                    : balance?.lt(amount)
-                    ? "Insufficient Balance"
```
Comment (madwiki) [inline src/views/Bridge/BridgePanel.tsx:1182]: why remove the "Insufficient Balance" tip?

**Review by madwiki — COMMENTED** [review]
Basically no problem, but the tip of Insufficient balance may be restored.

File: src/views/Bridge/BridgePanel.tsx (hunk 2 above)
Comment (chezhe) [reply src/views/Bridge/BridgePanel.tsx:1182]: > why remove the "Insufficient Balance" tip?

It's not tip, it's button text. This validation will be shown when user submit.

Comment (chezhe) [reply src/views/Bridge/BridgePanel.tsx:1182]: The old code make the button too complex. Now put all validation to the submit step.

**Pull request merged**